src/emc/usr_intf/gmoccapy/player.py

```diff
@@ -24,26 +24,36 @@
 
 '''
 import gi
-from gi.repository import Gtk as gtk
-from gi.repository import GObject as gobject
-import gst
+from gi.repository import GLib
+
+from qtvcp import logger
+LOG = logger.getLogger(__name__)
+# Force the log level for this module
+LOG.setLevel(logger.INFO) # One of DEBUG, INFO, WARNING, ERROR, CRITICAL# attempt to setup audio
```

Comment:
'LOG.setLevel'  is usually commented out.
It is used to force this file to a different log level then gmoccapy proper. Usually when debugging.

Reply:
Fixed.